src/interfaces/HposInterface.ts

```diff
@@ -23,9 +23,36 @@ interface HposInterface {
   getUnpaidInvoices: () => Promise<HposHolochainCallResponse>
   getRedemptionHistory: () => Promise<HposHolochainCallResponse>
   getCoreAppVersion: () => Promise<CoreAppVersion>
+  redeemHoloFuel: (payload: RedeemHoloFuelPayload) => Promise<HposHolochainCallResponse>
   HPOS_API_URL: string
 }
 
+interface RedeemHoloFuelPayload {
+  amount: string
+  note: string
+}
+
+export interface RedemptionTransaction {
+  id: string
+  amount: number
+  fee: string
+  completed_date: number
+  created_date: number
+  transaction_type: string
+  counterparty: string
+  direction: string
+  status: string
+  note?: string // The `eth_hash` will be stored here in Redemption Transactions
```

Comment:
```suggestion
  note?: string
  proof_of_service?:  {
    redemption: string, // The `eth_hash` (ie: wallet_address) will be stored here in Redemption Transactions
  }, 
```

Updating to match implementation changes.